sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -95,6 +102,11 @@
   /** The sessions dir. size for capping the number of envelopes Default is 100 */
   private int sessionsDirSize = 100;
 
+  private int maxQueueSize = cacheDirSize + sessionsDirSize;
+  // TODO: revert, this is necessary because start/end session + events will be higher than the
```

Comment:
I'm not sure I follow this comment. This maxQueueSize is not a single valid used by any file queue?
btw there are some changes that we'll be doing which will mean a single queue of envelopes to be submitted. That's to address the session updates vs events sync. So going forward we'll likely just merge and have a single folder

Reply:
our thread/transport queue is 10 items and it drops all the next threads that will try to be enqueued.
as I was doing a test with 100 sessions in a for loop, without a proper delay between them, I needed to increase the queue size.
the comment is just to UNDO it, this most probably is not a real use case, and the queue size is a param anyway, prople can increase if they want to

Reply:
Got it. Makes sense now, thanks for clarifying.